src/Disyuntor.ts

```diff
@@ -75,7 +77,7 @@ export class Disyuntor extends EventEmitter {
       );
     } else if (state === State.HalfOpen) {
       this.currentCooldown = Math.min(
-          this.currentCooldown * (this.failures + 1),
+          this.currentCooldown * (this.trips + 1),
```

Comment:
Can we say that `this.trips` is equivalent to `this.failures - this.config.thresholdConfig.maxConsecutiveFailures`? ie. the nb of failures _after_ reaching the threshold?

I wonder if it would be simpler to use that expression instead of introducing a new variable. wdyt?

Reply:
Yes you're right. Nice idea. Will update that.